<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenconnectProp</class>
 <widget class="QWidget" name="OpenconnectProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>518</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OpenConnect Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grp_general">
     <property name="title">
      <string comment="like in General settings">General</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Gateway:</string>
        </property>
        <property name="buddy">
         <cstring>leGateway</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leGateway"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>CA Certificate:</string>
        </property>
        <property name="buddy">
         <cstring>leCaCertificate</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KUrlRequester" name="leCaCertificate">
        <property name="filter">
         <string>*.pem *.crt *.key</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Proxy:</string>
        </property>
        <property name="buddy">
         <cstring>leProxy</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leProxy"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>CSD Wrapper Script:</string>
        </property>
        <property name="buddy">
         <cstring>leCsdWrapperScript</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="chkAllowTrojan">
        <property name="text">
         <string>Allow Cisco Secure Desktop trojan</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="KUrlRequester" name="leCsdWrapperScript"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>VPN Protocol:</string>
        </property>
        <property name="buddy">
         <cstring>cmbProtocol</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbProtocol">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Cisco AnyConnect</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Juniper Network Connect</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PAN Global Protect</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pulse Connect Secure</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Reported OS:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cmbReportedOs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GNU/Linux</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GNU/Linux 64-bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Windows</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mac OS X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Android</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Apple iOS</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Certificate Authentication</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>User Certificate:</string>
        </property>
        <property name="buddy">
         <cstring>leUserCert</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="leUserCert">
        <property name="filter">
         <string>*.pem *.crt *.key</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Private Key:</string>
        </property>
        <property name="buddy">
         <cstring>leUserPrivateKey</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="leUserPrivateKey">
        <property name="filter">
         <string>*.pem *.crt *.key</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="chkUseFsid">
        <property name="text">
         <string>Use FSID for key passphrase</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="preventInvalidCert">
        <property name="text">
         <string>Prevent user from manually accepting invalid certificates</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>278</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buTokens">
        <property name="text">
         <string>Token Authentication</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>enableTokenSecret(int)</slot>
 </slots>
</ui>
